src/app/home-page/widget/organization-box/organization-box.component.scss

```diff
@@ -0,0 +1,57 @@
+@use '@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.org-box {
+  flex: 1 1 0px;
```

Comment:
A lot of this flex styling can be achieved using `fxFlex` in the html file that uses these components (I think it's `newdashboard.component.html`. It would remove the need for these flex styles